libs/sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -1999,6 +1999,8 @@ public void test_create_role_statement() {
         printStatement("create role admin with inherit");
         printStatement("create role admin inherit");
         printStatement("create role admin InheRit");
+
+        printStatement("create role admin inherit login");
```

Comment:
In the first iteration I didn't spot a bug with using commas in formatter since we had validation which kicked in before formatter. Now validation is supposed to be done in the analyzer and this bug was hit.